lib/internal/fixed_queue.js

```diff
@@ -60,7 +60,7 @@ class FixedCircularBuffer {
   constructor() {
     this.bottom = 0;
     this.top = 0;
-    this.list = new Array(kSize);
+    this.list = new Array(kSize).fill(undefined);
```

Comment:
Does this need to be replaced with a primordial?

Reply:
It does, with `ArrayPrototypeFill(new Array(kSize), undefined)` importing it from primordials like Array above